src/hydra-eval-jobs/hydra-eval-jobs.cc

```diff
@@ -157,7 +157,8 @@ static void worker(
                 if (drv->querySystem() == "unknown")
                     throw EvalError("derivation must have a 'system' attribute");
 
-                auto drvPath = drv->queryDrvPath();
+                auto localStore = state.store.dynamic_pointer_cast<LocalFSStore>();
+                auto drvPath = localStore->printStorePath(drv->requireDrvPath());
```

Comment:
`localStore` might be `null`, please do not do this way.
```suggestion
                auto drvPath = state.store->printStorePath(drv->requireDrvPath());
```